### PR TITLE
support multiple puppetdb servers

### PR DESCRIPTION
**Pull request by h0tw1r3**
migrated from PR #351 originally submitted by @bassonj

per documentation at https://puppet.com/docs/puppetdb/7/puppetdb_connection.html#server_urls

### CODE REVIEW

**Comment by h0tw1r3** [comment]
I don't really own this per-say, nor have I tested it. The original PR was abandoned, and I was unable to rebase as the author's repo was migrated into another org.

**Comment by h0tw1r3** [comment]
Seems to address #409 but needs work for anyone who wants to pick it up.

**Comment by h0tw1r3** [comment]
Seems to address #386 but needs to be finished. Original author is no longer working on it if someone else is interested in picking this up.

File: manifests/database/ssl_configuration.pp [hunk 1]
```diff
@@ -6,7 +6,7 @@
   $database_username           = $puppetdb::params::database_username,
   $read_database_username      = $puppetdb::params::read_database_username,
   $read_database_host          = $puppetdb::params::read_database_host,
-  $puppetdb_server             = $puppetdb::params::puppetdb_server,
+  Variant[String,Array[String, 1]] $puppetdb_server = $puppetdb::params::puppetdb_server,
```
Comment (bastelfreak) [inline manifests/database/ssl_configuration.pp:9]: I think that's the least we should do. would even be better to enforce IP/FQDN, but I don't know out of my head what kind of values are allowed here.
```suggestion
  Variant[String[1],Array[String[1], 1]] $puppetdb_server = $puppetdb::params::puppetdb_server,
```

**Pull request closed without merging**